pages/index.vue

```diff
@@ -34,11 +34,11 @@ const { data: workshops } = useAsyncData('latest-workshops', () => queryContent(
   <div>
     <AppSection>
       <div class="md:flex md:flex-row items-center">
-        <div class="mt-8 md:mt-0 md:w-3/5 md:flex-1">
+        <div class="mt-8 md:mt-0 md:w-3/5 md:flex-1 md:flex-grow-[1.05] lg:flex-grow-[1.05]">
```

Comment:
```suggestion
        <div class="mt-8 md:mt-0 md:w-3/5 md:flex-1 md:flex-grow-[1.05]">
```

This would be enough as it defines it for "medium and bigger" screens